_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Introducing Security Analytics"
+authors:
+  - christophermoore
+date:   2023-03-13 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: security analytics
+meta_description: Provides and overview of Security Analytics and how it fits in with the OpenSearch Project
+---
+
+# Introducing Security Analytics
+
+In January of 2023, one of the largest wireless carriers in the US revealed that hackers had compromised a company database of customer information through unauthorized use of an API and had acquired the personal details of as many as 37 million individuals, including names, addresses, phone numbers, and account numbers. Once identified, the company was able to halt the malicious activity within a day. However, investigations indicated that the data breach had likely occurred at the end of November the previous year, giving the hackers over a month to extract the information.
```

Comment:
Could we just say "One of the largest wireless carriers in the US revealed recently, that hackers had ..." . This leads better than with the date.

Reply:
We can remove this line, we we've already drawn the attention of the reader. "However, investigations indicated that the data breach had likely occurred at the end of November the previous year, giving the hackers over a month to extract the information."